metric/bigtable/src/main/java/com/spotify/heroic/metric/bigtable/BigtableMetricModule.java

```diff
@@ -49,17 +55,18 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+@SuppressWarnings({"LineLength"})
 @ModuleId("bigtable")
 public final class BigtableMetricModule implements MetricModule, DynamicModuleId {
 
     private static final Logger log = LoggerFactory.getLogger(BigtableMetricModule.class);
     private static final String BIGTABLE_CONFIGURE_PARAM = "bigtable.configure";
 
     /* default number of Cells for each batch mutation */
-    public static final int DEFAULT_MUTATION_BATCH_SIZE = 10_000;
+    public static final int DEFAULT_MUTATION_BATCH_SIZE = 1_000;
 
     /* maximum possible number of Cells for each batch mutation */
-    public static final int MAX_MUTATION_BATCH_SIZE = 1_000_000;
+    public static final int MAX_MUTATION_BATCH_SIZE = 100_000;
```

Comment:
Reviewers: note that this had to be reduced lest the max batch size calls time out.

Reply:
Oh, finally, this kind of commit was called a “boy scout commit” on my last project. It’s basically an ethos where you try to make a source file that you need to touch, better than how you found it. Like organic refactoring sort of. Google do it and all that. 
But I do get that not everyone loves `var` so “better” is always subjective.